CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 1.7.0 (July 31, 2024)
+
+### Added
+- Add conversion from `Bytes` to `BytesMut` (#695, #710)
```

Comment:
The other entries have empty lines after the header. Can you add them for consistency?

Reply:
Fixed!